public/.evergreen.yml

```diff
@@ -61,14 +61,15 @@ functions:
           <<: *go_env
           MACOS_NOTARY_KEY: ${macos_notary_keyid}
           MACOS_NOTARY_SECRET: ${macos_notary_secret}
+        # TODO: triggered_by_git_tag is not set, because we run the task manually
```

Comment:
follow-up ticket: the goal here would be to merge public .evergreen with global .evergreen which should unify tag handling

Reply:
https://jira.mongodb.org/browse/CLOUDP-318631